src/components/Buttons/ArrowButton.tsx

```diff
@@ -18,6 +18,7 @@ const ArrowButton: React.FC<ArrowButtonProps> = ({
   return (
     <button
       aria-labelledby={arrowLabelledBy}
+      className="arrow-button"
```

Comment:
Why does this not have `arrow__hover--right-small` like the rest?

Reply:
What do you mean like the rest? the arrow__hover--right-small is from my understanding only used on the card element containing the arrow inside it.

Reply:
Just sharing my thoughts, and wanted to make sure it was not an oversight. Carry on good sir 😁